src/components/ui/dialog/custom-dialog.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react"
+import Dialog from "@components/ui/dialog/dialog"
+
+interface CustomDialogProps {
+  open: boolean
+  title: React.ReactNode
+  description: React.ReactNode
+  actions: React.ReactNode
+  variant?: "white" | undefined
+  size?: "small" | "medium" | undefined
+  maxWidthMin?: true | undefined
+}
+
+export const CustomDialog = ({
+  open,
+  title,
+  description,
+  actions,
+  variant,
+  size,
+  maxWidthMin,
+}: CustomDialogProps) => {
+  return (
+    <Dialog open={open} variant={variant} size={size} maxWidthMin={maxWidthMin}>
+      <Dialog.Title>{title}</Dialog.Title>
+      <Dialog.Description>{description}</Dialog.Description>
+      <Dialog.Actions>{actions}</Dialog.Actions>
```

Comment:
can we add this here and just have additional props

            <Button variant='primaryOutline' onClick={onClose}>
              No
            </Button>
            <Button variant='primary' onClick={onSubmit}>
              Yes
            </Button>

Reply:
possible props may include:
showCloseButton
showSubmitButton
closeButtonLabel
submitButtonLabel
onClose
onSubmit